community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/planner/logical/plans/rewriter/unnestOptional.scala

```diff
@@ -31,13 +31,13 @@ case object unnestOptional extends Rewriter {
   private val instance: Rewriter = Rewriter.lift {
     case apply@Apply(lhs,
       Optional(
-      e@Expand(_: Argument, _, _, _, _, _, _))) =>
+      e@Expand(_: Argument, _, _, _, _, _, ExpandAll))) =>
```

Comment:
This is something that maybe needs to be discussed, but I don't believe this is safe to do when `ExpandInto`